presto-main/src/main/java/com/facebook/presto/cost/RuntimePlanNodeStats.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cost;
+
+import com.facebook.presto.spi.statistics.RuntimeSourceInfo;
+import org.pcollections.HashTreePMap;
+
+import java.util.Arrays;
+
+// Represents stats collected during execution of the query.
+public class RuntimePlanNodeStats
```

Comment:
We don't yet have a need for bytesByPartitionId, and it's not clear to me whether this is the right thing to do when we will eventually want to use it (skew join).  I would just use basic PlanNodeStateEstimate directly for now.  Then when we want to actually use the bytesByPartitionId we can revisit.